frontend/src/pages/Home/NavigationButton.jsx

```diff
@@ -1,15 +1,18 @@
 import PropTypes from "prop-types";
 import { useNavigate } from "react-router-dom";
-import "./NavigationButton.scss";
 
-function NavigationButton({ text, destination }) {
+function NavigationButton({ text, destination, className }) {
   const navigate = useNavigate();
 
   const handleButtonClick = () => {
     navigate(destination);
   };
   return (
-    <button className="style-button" type="button" onClick={handleButtonClick}>
+    <button
+      className={`style-button, style-button2, style-button3 ${className}`}
```

Comment:
there is a variable className, used as a props, hard coded class still usefull ?

Reply:
Absolutely, there are different styles of buttons, so one class per button

Reply:
Modified passed with props